frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -483,15 +487,7 @@ export function PlotlyChart({
               }
             : undefined
         }
-        onInitialized={figure => {
-          widgetMgr.setElementState(element.id, "figure", figure)
-        }}
-        // Update the figure state on every change to the figure itself:
-        onUpdate={figure => {
-          // Save the updated figure state to allow it to be recovered
-          widgetMgr.setElementState(element.id, "figure", figure)
-          setPlotlyFigure(figure)
-        }}
+        onInitialized={() => {}}
```

Comment:
I think removing this part will break some existing features e.g., selections won't be kept in the UI if the chart gets unmounted, see the changed snapshots.